PluginImports.targets

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <BepInExDir>$(DSPGameDir)BepInEx\core\</BepInExDir>
     <DSPAssemblyDir>$(DSPGameDir)DSPGAME_Data\Managed\</DSPAssemblyDir>
+    <PluginOutputDirectory>$(DSPGameDir)BepInEx\plugins\Nebula\</PluginOutputDirectory>
```

Comment:
What exactly is this? I couldn't find a use of this anywhere and google doesn't give me much to go on

Reply:
Oh, it's stuff I haven't committed yet, my bad.

Want to use the prop in the copy to game dir targets.

Reply:
Fixed.